### PR TITLE
chore: Clean up classes, dependencies and naming

### LINKED ISSUE #124: Remove redundant classes and cleanup
Due to so many major refactoring, we have some classes which are unused and some test parameters too.

This should be cleaned up

I'm working on it

### PR DESCRIPTION
**Pull request by iamareebjamal**
Fixes #124 

- Remove unneeded test initializations
- Remove unused classes
- Remove unused dependencies
- Normalize naming of classes
- Remove personal artifacts from strings

### CODE REVIEW

File: app/src/main/java/org/fossasia/openevent/app/utils/Constants.java [hunk 1]
```diff
@@ -33,6 +33,6 @@
     //attendee scanned index
     public static final String SCANNED_INDEX = "index";
     //No network available string
-    public static final String NO_NETWORK = "Network Not Available";
+    public static final String NO_NETWORK = "NetworkUtils Not Available";
```
Comment (mejariamol) [inline app/src/main/java/org/fossasia/openevent/app/utils/Constants.java]: `Network not available` is more user friendly.

Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/app/utils/Constants.java]: Sorry, naming error in refactoring 

Thanks for heads up

Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/app/utils/Constants.java]: Fixed

**Comment by iamareebjamal** [comment]
@niranjan94 Please merge

**Pull request merged**